### PR TITLE
Update how-to-prerequisites.md

### PR DESCRIPTION
**Pull request by baardhermansen**
Removed all tilde from URLs. 
Updated the registry key for TLS so it's easier to implement.
Several small changes to conform to markdown style.
Removed several double spaces (` `) and trailing spaces.

Could not find any references in Microsoft documentation to the tabulated table, so didn't dare touch the formatting.
There are what appear to be missing trailing pipes in both tables, and several extra line shifts though, so might need some more cleaning up.

### CODE REVIEW

**Comment by Jak-MS** [comment]
@billmath 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by Jak-MS** [comment]
@billmath @baardhermansen - I'm 99% sure that the changes to the links (eg. removing the \~ and in a couple of cases adding \.md) will break the links. Those aren't working in the github view (and I wouldn't expect them to), but validation isn't catching them. If you are confident these links will work, the commits in this PR should be moved to the private repo for a full review with staging. Let us know if you need help with that. thanks.

**Comment by Jak-MS** [comment]
#hold-off

**Comment by baardhermansen** [comment]
> @billmath @baardhermansen - I'm 99% sure that the changes to the links (eg. removing the ~ and in a couple of cases adding .md) will break the links. Those aren't working in the github view (and I wouldn't expect them to), but validation isn't catching them. If you are confident these links will work, the commits in this PR should be moved to the private repo for a full review with staging. Let us know if you need help with that. thanks.

@Jak-MS 
My source for removing the tilde and some places adding .md: https://learn.microsoft.com/en-us/contribute/content/how-to-write-links#links-from-one-article-to-another

I've never moved a PR before, so if you could point me to an article (if such exists) or give me instructions, i'll gladly move :)

**Comment by Jak-MS** [comment]
@baardhermansen @billmath - i'll move the commits and report back here :)  it might take me a day.

**Comment by baardhermansen** [comment]
@Jak-MS Thanks, i'm in no rush, take your time ;)

**Comment by Jak-MS** [comment]
I moved the commits to the private repo `entra-docs-pr` and opened a PR there. Validation threw some errors as shown below:
image

I was able to fix this by adding back the tilde and the .md to the respective links. (which I will do to this PR so it can be merged)

@MicrosoftDocs/public-repo-pr-review-team @CelesteDG - **Noting this because it seems like the validation rules are different in the public repo (and do not flag broken links).**

File: docs/identity/hybrid/cloud-sync/how-to-prerequisites.md [hunk 1]
```diff
@@ -13,69 +13,70 @@ ms.author: billmath
...
 
-### Prerequisites for gMSA:
+A group Managed Service Account is a managed domain account that provides automatic password management, simplified service principal name (SPN) management, the ability to delegate the management to other administrators, and also extends this functionality over multiple servers. Microsoft Entra Cloud Sync supports and uses a gMSA for running the agent. You will be prompted for administrative credentials during setup, in order to create this account. The account will appear as `domain\provAgentgMSA$`. For more information on a gMSA, see [group Managed Service Accounts](/windows-server/security/group-managed-service-accounts/group-managed-service-accounts-overview).
+
+### Prerequisites for gMSA
+
 1. The Active Directory schema in the gMSA domain's forest needs to be updated to Windows Server 2012 or later.
-2. [PowerShell RSAT modules](/windows-server/remote/remote-server-administration-tools) on a domain controller
+2. [PowerShell RSAT modules](/windows-server/remote/remote-server-administration-tools.md) on a domain controller.
```
Comment (Jak-MS) [inline docs/identity/hybrid/cloud-sync/how-to-prerequisites.md:36]:
```suggestion
2. [PowerShell RSAT modules](/windows-server/remote/remote-server-administration-tools) on a domain controller.
```

File: docs/identity/hybrid/cloud-sync/how-to-prerequisites.md [hunk 2]
```diff
@@ -13,69 +13,70 @@ ms.author: billmath
...
-
 ### In the Microsoft Entra admin center
 
-1. Create a cloud-only hybrid identity administrator account on your Microsoft Entra tenant. This way, you can manage the configuration of your tenant if your on-premises services fail or become unavailable. Learn about how to [add a cloud-only hybrid identity administrator account](~/fundamentals/add-users.md). Finishing this step is critical to ensure that you don't get locked out of your tenant.
-1. Add one or more [custom domain names](~/fundamentals/add-custom-domain.md) to your Microsoft Entra tenant. Your users can sign in with one of these domain names.
+1. Create a cloud-only hybrid identity administrator account on your Microsoft Entra tenant. This way, you can manage the configuration of your tenant if your on-premises services fail or become unavailable. Learn about how to [add a cloud-only hybrid identity administrator account](/fundamentals/add-users.md). Finishing this step is critical to ensure that you don't get locked out of your tenant.
+1. Add one or more [custom domain names](/fundamentals/add-custom-domain.md) to your Microsoft Entra tenant. Your users can sign in with one of these domain names.
```
Comment (Jak-MS) [inline docs/identity/hybrid/cloud-sync/how-to-prerequisites.md:62]:
```suggestion
1. Create a cloud-only hybrid identity administrator account on your Microsoft Entra tenant. This way, you can manage the configuration of your tenant if your on-premises services fail or become unavailable. Learn about how to [add a cloud-only hybrid identity administrator account](~/fundamentals/add-users.md). Finishing this step is critical to ensure that you don't get locked out of your tenant.
1. Add one or more [custom domain names](~/fundamentals/add-custom-domain.md) to your Microsoft Entra tenant. Your users can sign in with one of these domain names.
```

File: docs/identity/hybrid/cloud-sync/how-to-prerequisites.md [hunk 3]
```diff
@@ -13,69 +13,70 @@ ms.author: billmath
...
 
 ### In your directory in Active Directory
 
-Run the [IdFix tool](/microsoft-365/enterprise/set-up-directory-synchronization) to prepare the directory attributes for synchronization.
+Run the [IdFix tool](/microsoft-365/enterprise/set-up-directory-synchronization.md) to prepare the directory attributes for synchronization.
```
Comment (Jak-MS) [inline docs/identity/hybrid/cloud-sync/how-to-prerequisites.md:66]:
```suggestion
Run the [IdFix tool](/microsoft-365/enterprise/set-up-directory-synchronization) to prepare the directory attributes for synchronization.
```

**Comment by baardhermansen** [comment]
@Jak-MS There's something strange about the verification of links indeed. 
In the article [i referenced above](https://learn.microsoft.com/en-us/contribute/content/how-to-write-links#links-from-one-article-to-another), there's a Notes section that states 

> To link to an absolute path that begins at the root of the repository, start the link with /. Including the ~/ produces invalid links when navigating the source repositories on GitHub. Starting the path with / resolves correctly.

**Comment by Jak-MS** [comment]
@baardhermansen - I see that ... and that note is a little confusing, I think. Starting a link with / is usually for links to articles in another Microsoft Learn repo.  I'll reach out to my team about this.

**Pull request merged**